package.json

```diff
@@ -51,13 +52,19 @@
   },
   "main": "node_modules/docpad/bin/docpad-server",
   "scripts": {
-    "start": "npm run build && docpad generate && gulp minify && node server.js",
-    "local": "gulp start -f && gulp getDependencyInteractiveDocu && docpad generate && gulp minify && node server.js",
-    "production": "npm install && bower install && gulp start && gulp getDependencyInteractiveDocu && docpad generate --env prod && gulp minify",
-    "build": "npm install && bower install -f && npm run build:modernizr && npm run build:zeroclipboard && npm run build:lunrplugin && npm run build:jws && gulp start -f && gulp getDependencyInteractiveDocu",
-    "build:modernizr": "cp ./src/raw/bower_components/modernizr/modernizr.js ./src/raw/scripts/general",
-    "build:jws": "mkdir -p ./src/raw/scripts/vendor && cp ./src/raw/bower_components/kjur-jsrsasign/jws-3.3.js ./src/raw/scripts/vendor",
-    "build:zeroclipboard": "cp ./src/raw/bower_components/zeroclipboard/dist/ZeroClipboard.js ./src/raw/scripts/general && cp ./src/raw/bower_components/zeroclipboard/dist/ZeroClipboard.swf ./src/raw/scripts/general",
-    "build:lunrplugin": "npm install --prefix ./plugins/docpad-plugin-lunr"
+    "install-deps": "npm install && npm run install:bower",
```

Comment:
Because it is not part of something, this is just a word, there are multiple scripts concerning build so I added this prefix, `install-deps` task is installing all dependencies